config.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
+)
+
+func getConfig() MetadataS3Config {
+	parseConfig()
+	S3Conf := configS3Storage()
+	return S3Conf
+
+}
+
+type MetadataS3Config struct {
+	URL               string
+	Port              int
+	AccessKey         string
+	SecretKey         string
+	Bucket            string
+	Region            string
+	Chunksize         int
+	Cacert            string
+	WebMetadataFolder string
+}
+
+func configS3Storage() MetadataS3Config {
+	s3 := MetadataS3Config{}
+	s3.URL = viper.GetString("S3MetadataBucket.url")
+	s3.AccessKey = viper.GetString("S3MetadataBucket.accesskey")
+	s3.SecretKey = viper.GetString("S3MetadataBucket.secretkey")
+	s3.Bucket = viper.GetString("S3MetadataBucket.bucket")
+	s3.Port = 9000
```

Comment:
Put in config?

Reply:
It is picking the values from the config, except port.